packages/components-js/tests/e2e/specs/accordion.e2e.ts

```diff
@@ -58,6 +58,8 @@ const getBody = (page: Page) => page.$('body');
 const getCollapseVisibility = async (page: Page) => getElementStyle(await getCollapsible(page), 'visibility');
 const getCollapseGridTemplateRows = async (page: Page) =>
   getElementStyle(await getCollapsible(page), 'gridTemplateRows');
+const getHeadingTagName = async (page: Page): Promise<string> =>
```

Comment:
Should be able to pierce through shadow dom boundaries with `$` selector. Also Playwright encourages to use [Locators](https://playwright.dev/docs/locators).

Reply:
I have abstracted the test and fixed it.